src/components/elements/modal.tsx

```diff
@@ -43,7 +43,8 @@ const HeaderContainer = styled(Flex)`
 `
 
 const ContentContainer = styled.div`
-    overflow: scroll;
+    overflow: auto;
+    max-height: 70vh;
```

Comment:
Do we need this?

Reply:
yes. with `overflow: scroll;` there will be a disabled scrollbar (both on x and y) in linux devices (and maybe windows) even if the content is not big . So I added this to show the scrollbar only if the content is too much that it needs more than 70vh height.